metrics/prometheus/prometheus.yml.tpl

```diff
@@ -190,3 +190,21 @@ done)
       - targets: ['ci.ipfs.team']
         labels:
           group: jenkins
+
+  - job_name: 'dokku - docker'
+    scheme: http
+    metrics_path: '/metrics'
+    honor_labels: false
+    static_configs:
+      - targets: ['cloud.ipfs.team:9323']
+        labels:
+          group: dokku
+
+  - job_name: 'dokku - cadvisor'
+    scheme: http
+    metrics_path: '/metrics'
+    honor_labels: false
+    static_configs:
+      - targets: ['cloud.ipfs.team:9324']
```

Comment:
Is it a good idea to run these metrics publicly?

Reply:
As far as I can tell after looking through the metrics, is that there is no information we have to keep secret in those metrics. Also, if we have sensitive services, they should not be run on cloud.ipfs.team but rather their own instance, our little dokku host is becoming a snowflake in general.